drud/ddev
# [FEATURE] Allow an optional path for the launch command

## The Problem/Issue/Bug:
Sometime the user likes to open a specific path of the website instead of the entry point.

## How this PR Solves The Problem:
This patch allows the user to provide an additional path for the launch command to directly head to the defined path of the website instead of the root entry point.

## Manual Testing Instructions:
1. Run `ddev launch mypath` to see that the browser opens the correct URL. Replace `mypath` by the path you like to open.
2. Run `ddev launch` to see that the browser opens the root entry point of the website as before.

## Automated Testing Overview:

## Related Issue Link(s):

## Release/Deployment notes:
